### DISCUSSION TITLE
Bar Border for Healthbar

### DISCUSSION THREAD

**Discussion by NotGodfall** [Backupiseasy/ThreatPlates#537] [Ideas]
Would love to see a Bar Border selector just like in the "Resource" tab but for the Health bar.
I'm not a fan of a simple solid border or having no borders at all, this would be a great addition for people who love customization.

Let me know if this is impossible!

**Comment by Backupiseasy** [comment]
Yeah, that's also something I want to implement for some time, but I have currently lined up so much that I am not sure when this will happen. 

It's not that difficult, BTW, as the basic statusbar is already there (and used for the resource bar, for example). But it requires some changes in the options.
